pkg/sensors/tracing/generickprobe.go

```diff
@@ -552,6 +551,23 @@ func getKprobeSymbols(symbol string, syscall bool, lists []v1alpha1.ListSpec) ([
 	return []string{symbol}, syscall, nil
 }
 
+type hasMaps struct {
+	stackTrace bool
+	rateLimit  bool
+	fdInstall  bool
+}
+
+func hasMapsSetup(spec *v1alpha1.TracingPolicySpec) *hasMaps {
+	has := &hasMaps{}
+	for _, kprobe := range spec.KProbes {
+		has.fdInstall = has.fdInstall || selectorsHaveFDInstall(kprobe.Selectors)
+		if has.fdInstall {
+			break
+		}
```

Comment:
you could simplify it, we can always modify if this gets more complicated later
```suggestion
		if selectorsHaveFDInstall(kprobe.Selectors) {
			has.fdInstall = true
			break
		}
```
or
```suggestion
		if selectorsHaveFDInstall(kprobe.Selectors) {
			return &hasMap{
				fdInstall: true,
			}
		}
```